backend/hint/builtin.go

```diff
@@ -10,7 +10,7 @@ var (
 	// IsZero computes the value 1 - a^(modulus-1) for the single input a. This
 	// corresponds to checking if a == 0 (for which the function returns 1) or a
 	// != 0 (for which the function returns 0).
-	IsZero = NewStaticHint(isZero)
+	IsZero = isZero
```

Comment:
I initially thought that it would be great to be able to collect all the built-in hints into a single `var` block for easier reference. Now there is only a single built-in hint left and now this doesn't make too much sense anymore. Maybe move the comment to `isZero` function and make that public instead?